javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,120 @@
+/* My age in 2027*/ 
+let yearOfBerth;
+let yearFuture;
+yearOfBerth = 1985; // integer, 
+yearFuture = 2027; // integer, 
+const age = yearFuture - yearOfBerth;
+console.log('You will be' + age+ 'years old in' + yearFuture);
+
+/* Dog age*/
+let dogYearOfBirth;
+let dogYearFuture;
+dogYearOfBirth = 2017;
+dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false;
+if (shouldShowResultInDogYears === true)
```

Comment:
When you are checking if a variable is true, doing 
``` if (shouldHowResultInDogYears) ``` is the same as
 ``` if (shouldHowResultInDogYears === true)```
It look cleaner, and usually the less code you need to write the better ;)